packages/docusaurus-plugin-content-blog/src/pluginOptionSchema.ts

```diff
@@ -20,6 +20,7 @@ export const DEFAULT_OPTIONS = {
   blogTagsListComponent: '@theme/BlogTagsListPage',
   blogPostComponent: '@theme/BlogPostPage',
   blogListComponent: '@theme/BlogListPage',
+  blogDescription: '',
```

Comment:
```suggestion
  blogDescription: 'Blog',
```

Let's keep it retrocompatible, it displays Blog by default currently